### PR TITLE
Implement null check on performCompletion

### PR DESCRIPTION
**Pull request by slmyldz**

### CODE REVIEW

File: library/src/main/java/com/tokenautocomplete/TokenCompleteTextView.java [hunk 1]
```diff
@@ -470,7 +470,7 @@ public boolean enoughToFilter() {
 
     @Override
     public void performCompletion() {
-        if (getListSelection() == ListView.INVALID_POSITION && enoughToFilter()) {
+        if (getListSelection() == ListView.INVALID_POSITION && enoughToFilter() && getAdapter() != null) {
             Object bestGuess;
             if (getAdapter().getCount() > 0 && performBestGuess) {
```
Comment (mgod) [inline library/src/main/java/com/tokenautocomplete/TokenCompleteTextView.java]: Should probably do the null check on this line instead. Don't want to disable defaultObject completions fir everyone using a custom list implementation.

**Comment by slmyldz** [comment]
Maybe some developers use custom list implementation and got error that line. They can find how to fix it there.

**Comment by mgod** [comment]
Ah, sorry, I had thought the crash hit on line 475. The crash you're running into is at line 475?

**Comment by mgod** [comment]
I'm going to go ahead and implement a slightly more nuanced fix to this and release it this afternoon.

**Comment by mgod** [comment]
This should be fixed in 2.0.6

**Comment by slmyldz** [comment]
thank you :+1:

**Pull request closed without merging**